src/mex.jl

```diff
@@ -0,0 +1,52 @@
+export mex_init, mex_args, mex_return, mex_showerror, mex_eval, mex_call
```

Comment:
I need to wrap this in a `MEX` module and get rid of the superfluous `mex_` prefix everywhere